samples/Basic YARP Sample/Properties/launchSettings.json

```
@@ -0,0 +1,14 @@
+﻿{
+  "$schema": "http://json.schemastore.org/launchsettings.json",
+  "profiles": {
+    "YARP Proxy Sample": {
+      "commandName": "Project",
+      "launchBrowser": true,
```

With the Urls set in the appsettings.json rather than the launchSettings.json, the tooling doesn't know which url to launch. I don't think this applies to the command line scenarios anyways.
```suggestion
      "launchBrowser": false,
```